patterns/events-three-columns-images-and-titles.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Title: Events three columns with event images and titles
+ * Slug: twentytwentyfive/events-three-columns-images-and-titles
+ * Categories: featured, query
+ * Description: A header with title and text and three columns that show 3 events with their images and titles.
+ * Keywords: events, columns, images
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"metadata":{"categories":["featured"],"patternName":"twentytwentyfive/events-three-columns-images-and-titles","name":"Events three columns with event images and titles"},"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"},"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide" style="padding-top:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50)"><!-- wp:group {"align":"wide","layout":{"type":"constrained","justifyContent":"left"}} -->
+<div class="wp-block-group alignwide"><!-- wp:heading -->
+<h2 class="wp-block-heading">Events</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>These are some of the upcoming events.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group -->
+
+<!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"0","left":"var:preset|spacing|50"},"padding":{"top":"0","bottom":"0"}}}} -->
+<div class="wp-block-columns alignwide" style="padding-top:0;padding-bottom:0"><!-- wp:column {"style":{"spacing":{"padding":{"top":"var:preset|spacing|70"},"blockGap":"0"}}} -->
+<div class="wp-block-column" style="padding-top:var(--wp--preset--spacing--70)"><!-- wp:image {"sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/image-from-rawpixel-id-8802835-jpeg-scaled.webp" alt="Event image"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:paragraph {"style":{"elements":{"link":{"color":{"text":"var:preset|color|primary"}}},"spacing":{"padding":{"top":"var:preset|spacing|40"}}},"textColor":"primary","fontSize":"medium"} -->
+<p class="has-primary-color has-text-color has-link-color has-medium-font-size" style="padding-top:var(--wp--preset--spacing--40)">September 2, 2024</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:heading {"level":3,"style":{"spacing":{"padding":{"top":"var:preset|spacing|20"}}}} -->
+<h3 class="wp-block-heading" style="padding-top:var(--wp--preset--spacing--20)">Tell your story</h3>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"style":{"spacing":{"padding":{"top":"var:preset|spacing|40"}}},"fontSize":"large"} -->
```

Comment:
`Large` is the default size for the paragraph, so it is not needed here. Remember to also remove the class name on the `<p> `tag below.

Reply:
Please remove it from all 3 paragraphs.